addons/binding/org.openhab.binding.vitaled/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="vitaled" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="vitaled">
+		<label>VitaLED Binding</label>
+		<description>Binding for Brumberg VitaLED. This binding uses the Brumberg VitaLED LAN master for controlling VitaLED zones.</description>
+		<channel-groups>
+			<channel-group id="zone1" typeId="zone" />
+			<channel-group id="zone2" typeId="zone" />
+			<channel-group id="zone3" typeId="zone" />
+			<channel-group id="zone4" typeId="zone" />
+			<channel-group id="zone5" typeId="zone" />
+			<channel-group id="zone6" typeId="zone" />
+			<channel-group id="zone7" typeId="zone" />
+			<channel-group id="zone8" typeId="zone" />
+		</channel-groups>
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>Network Address</label>
+				<description>The IP of the VitaLED LAN Master</description>
+				<default>192.168.0.40</default>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="port" type="integer" required="true" min="1" max="65335">
+				<label>Port</label>
+				<description>Port of the VitaLED LAN Master</description>
+				<default>80</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refreshInterval" type="integer">
+				<label>Refresh Interval</label>
+				<description>States how often a refresh shall occur in seconds.</description>
+				<default>60</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-group-type id="zone">
+		<label>VitaLED Zone</label>
+		<description>This represents a VitaLED zone</description>
+		<channels>
+			<!-- Achromatic Light and intensity -->
+			<channel id="achromaticLight" typeId="achromaticLight">
+				<label>Achromatic Light</label>
+				<description>Achromatic Light in Kelvin</description>
+			</channel>
+			<channel id="intensity" typeId="intensity">
+				<label>Intensity</label>
+				<description>Intensity in %</description>
+			</channel>
+			<!-- Chromatic Light - RGBW control -->
+			<channel id="red" typeId="redLight">
```

Comment:
Do you only want to give the users direct access to the raw RGBW channels or would you also like to provide a `HSBType` channel, so openHAB users can use a `ColorPicker` to select a color?